Gruntfile.js

```diff
@@ -450,8 +459,8 @@ module.exports = function( grunt ) {
 
   task.registerMultiTask("browserscopekeys", "aggregate embed browserscope keys", function() {
     // Gather config files, initialize program vars
-    var files = file.expandFiles( this.file.src ),
-        name = this.file.dest,
+    var files = file.expand( this.files[0].src ),
+        name = this.files[0].dest,
         browserscopekeys = Yaml.load( file.read( files[0] ) );
```

Comment:
```
diff --git a/Gruntfile.js b/Gruntfile.js
index 9fb8e74..dd11c8d 100644
--- a/Gruntfile.js
+++ b/Gruntfile.js
@@ -463,9 +463,6 @@ module.exports = function( grunt ) {
         name = this.files[0].dest,
         browserscopekeys = Yaml.load( file.read( files[0] ) );

-    // Wrap in an array
-    browserscopekeys = [ browserscopekeys ];
-
     file.write( name, "App.register( \"browserscopekeys\", " + JSON.stringify( browserscopekeys, null, 2 ) + ");" );

     // Otherwise, print a success message.
```